python-checks/src/test/resources/checks/hotspots/hashing/hashingData.py

```diff
@@ -113,23 +149,13 @@ def cryptodome():
     from Cryptodome.Hash import MD2
     from Cryptodome.Hash import MD4
     from Cryptodome.Hash import MD5
-    from Cryptodome.Hash import SHA
-    from Cryptodome.Hash import SHA224
-    from Cryptodome.Hash import SHA256
-    from Cryptodome.Hash import SHA384
-    from Cryptodome.Hash import SHA512
-    from Cryptodome.Hash import HMAC
+    from Cryptodome.Hash import SHA1
 
     Cryptodome.Hash.MD2.new() # Noncompliant
     MD2.new()                 # Noncompliant
     MD4.new()                 # Noncompliant
     MD5.new()                 # Noncompliant
-    SHA.new()                 # Noncompliant
-    SHA224.new()              # Noncompliant
-    SHA256.new()              # Noncompliant
-    SHA384.new()              # Noncompliant
-    SHA512.new()              # Noncompliant
-    HMAC.new(b"\x00")         # Noncompliant
```

Comment:
Testing that we don't raise an issue anymore on those cases could be useful. Same thing for PyCrypto.